.gitignore

```diff
@@ -97,6 +97,7 @@ data.ms/*
 auth.json
 
 /packages/ux-shared/
+/packages/data-provider/undefined/
```

Comment:
`/undefined/` What's the purpose for this change?

Reply:
sometimes when building the data-provider's packages, some random files were created under this directory

Reply:
yes I've noticed but they usually get cleared, I'll accept this then